DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/TutorialViewModelTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class TutorialViewModelTests
+    {
+        private const int CustomisationId = 1;
+        private const int SectionId = 10;
+
+        [Test]
+        public void Tutorial_should_have_tutorial_content()
+        {
+            // Given
+            var expectedCourseContent = TutorialContentHelper.CreateDefaultTutorialContent();
+
+            // When
+            var initialMenuViewModel = new TutorialViewModel(expectedCourseContent, CustomisationId, SectionId);

```

Comment:
Looks like this one still needs renaming? 🙂

Reply:
Whoops I thought I had fixed all of those! Sorted in 86f7166 🙂